src/content/structured/get-started/figma.mdx

```diff
@@ -52,6 +45,14 @@ The UI Kit is turned on by default for all projects within our Figma organisatio
   />
 </DoubleDoDontCaution>
 
-## How to use the Figma UI Kit components
+## Not part of the Intelligence Community?
+
+We've published our [Figma UI Kit](https://www.figma.com/community/file/1283021660431259186) to the Figma Community, so that everyone can use our resources.
```

Comment:
Do we need the comma?

'We've published our Figma UI kit to the Figma community so that everyone can use our resources.

Reply:
Also, push out link please.